frontend/public/components/api-explorer.tsx

```diff
@@ -339,48 +348,56 @@ const APIResourcesList = compose(
 });
 APIResourcesList.displayName = 'APIResourcesList';
 
-export const APIExplorerPage: React.FC<{}> = () => (
-  <>
-    <Helmet>
-      <title>Explore API Resources</title>
-    </Helmet>
-    <div className="co-m-nav-title">
-      <h1 className="co-m-pane__heading">Explore API Resources</h1>
-    </div>
-    <APIResourcesList />
-  </>
-);
+export const APIExplorerPage: React.FC<{}> = () => {
+  const { t } = useTranslation();
+  return (
+    <>
+      <Helmet>
+        <title>{t('api-explorer~Explore API Resources')}</title>
+      </Helmet>
+      <div className="co-m-nav-title">
+        <h1 className="co-m-pane__heading">{t('api-explorer~Explore API Resources')}</h1>
```

Comment:
Should "Resources" be lowercase here the way it is above?

Reply:
@rebeccaalpert this is a page title, so i think its capitalized.

Reply:
Perfect! Lgtm then.